Libs/MRML/Core/Testing/vtkMRMLVolumeArchetypeStorageNodeTest1.cxx

```diff
@@ -124,6 +126,64 @@ int TestVoxelVectorType(const std::string& tempDir, const std::string& fileExten
   return EXIT_SUCCESS;
 }
 
+
+int TestFlipsLeftHandedVolumes(const std::string& tempDir)
+{
+  std::cout << "TestFlipsLeftHandedVolumes" << std::endl;
+  // Create a flipped image data with arbitrary K values
+  vtkNew<vtkMRMLScene> scene;
+
+  auto volumeNode = vtkMRMLScalarVolumeNode::SafeDownCast(
+      scene->AddNewNodeByClass("vtkMRMLScalarVolumeNode"));
+  CHECK_NOT_NULL(volumeNode);
+
+  int n_i{10}, n_j{20}, n_k{30};
+  vtkNew<vtkImageData> imageData;
+  imageData->SetDimensions(n_i, n_j, n_k);
+  imageData->AllocateScalars(VTK_FLOAT, 1);
+  imageData->GetPointData()->GetScalars()->Fill(0);
+
+  for (int i = 0; i < n_i; i++) {
```

Comment:
seems coding style is not followed, `{` probably needs to be on a new line here and in the next loop

Reply:
Thx, I just reformatted the code.